ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -615,7 +628,7 @@ func (r *RayJobReconciler) initRayJobStatusIfNeed(ctx context.Context, rayJob *r
 		return nil
 	}
 
-	if rayJob.Status.JobId == "" {
+	if rayJob.Spec.SubmissionMode != rayv1.NoneMode && rayJob.Status.JobId == "" {
```

Comment:
Can you update the comment for this function?

Reply:
Done